vizro-core/src/vizro/models/_components/form/dropdown.py

```diff
@@ -115,18 +114,7 @@ def _build_dynamic_placeholder(self):
             _, default_value = get_options_and_default(self.options, self.multi)
             self.value = default_value
 
-        # TODO-NEXT: Replace this with the "universal Vizro placeholder" component.
-        return html.Div(
-            children=[
-                html.Legend(children=self.title, className="form-label") if self.title else None,
-                dmc.DateRangePicker(
-                    id=self.id,
-                    value=self.value,
-                    persistence=True,
-                    persistence_type="session",
-                ),
-            ]
-        )
+        return self.__call__(self.options)
```

Comment:
This is the main change that's described in the PR description

Reply:
I didn't review the initial PR on when this was introduced, but glad to see this go 😄 🥳